src/core_modules/capture-core/components/DataEntries/EnrollmentRegistrationEntry/hooks/getSubValueForTei.js

```diff
@@ -11,14 +13,8 @@ const getImageOrFileResourceSubvalue = async (key: string, querySingleResource:
 };
 
 const getOrganisationUnitSubvalue = async (key: string, querySingleResource: QuerySingleResource) => {
-    const organisationUnit = await querySingleResource({
-        resource: 'organisationUnits',
-        id: key,
-        params: {
-            fields: 'id,name',
-        },
-    });
-    return { ...organisationUnit };
+    const organisationUnit = await getOrgUnitNames([key], querySingleResource);
```

Comment:
It would be great if we can make a wrapper function called `getOrgUnitName` (singular).